### DISCUSSION TITLE
Where could I find more information about tailwindcss 3.0 browser support?

### DISCUSSION THREAD

**Discussion by gaohongwei** [tailwindlabs/tailwindcss#12066] [Help]
Our team considering using tailwind library in large B2B product. But we have concerns regarding browser support in Asian markets.

We could find information on Browser Support in official documentation. 
It mentions Chrome, Safari, Firefox, Edge. And tailwind css 3.0 doesn't support IE including IE11.

Where I could find information regarding Samsung Internet Browser, QQ Browser, Whale Browser and 360 Secure Browser (popular in China and South Korea).

**Comment by wongjn** [comment]
You could consider using [Can I Use](https://caniuse.com/) or [MDN documentation](https://developer.mozilla.org/) for reference for browser support for CSS features you use via Tailwind.
